.travis.yml

```diff
@@ -14,7 +14,7 @@ jdk:
   - oraclejdk8
 
 script:
-  - ./gradlew clean check -x checkstyleTest -x :apollo-gradle-plugin:test --stacktrace
+  - ./gradlew clean check -x checkstyleTest --stacktrace
```

Comment:
I'd prefer to run build rather than Check. This way if we have dexing errors they will be caught in CI